microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -43,17 +69,24 @@ To use the Microsoft Edge Add-ons API, you need to enable the API for your proje
 
 1. On the **Publish API** page, click the **Create API credentials** button.  This step may take a few minutes to finish.
 
-   The API credentials have now been created; you've enabled or renewed the API.  The **Client ID**, **Client secret**, **Expiry date**, and **Access token URL** are now displayed on the Publish APIs page:
+   The API credentials have now been created; you've enabled or renewed the API.  The following items are now displayed on the **Publish API** page:
+   * **Client ID**
+   * **Client secret** (for REST API v1) or **API key** (for REST API v2)
+   * **Expiry date**
+   * **Access token URL** (for REST API v1)
 
    ![The 'Publish API' page at Partner Center after clicking 'Create API credentials', now showing Client ID, Client Secret, and Auth Token URL](./using-addons-api-images/create-api-credentials-button.png)
 
-1. Write down the **Client ID**, **Client secret** and the **Access token URL**.  You'll use these values in the next step, to get an access token.
+1. Write down the **Client ID**, **Client secret(API key for v2)** and the **Access token URL(v1)**.  You'll use these values in the next step, to get an access token.
 
 > [!IMPORTANT]
-> Be sure to write down the client secret now, because it's only visible immediately after enabling or renewing the API (that is, after creating API credentials). This particular secret isn't shown again.
+> This applies only to this REST Publish API v1, not v2:
+> Be sure to write down the client secret now, because it's only visible immediately after enabling or renewing the API (that is, after creating API credentials).  This particular secret isn't shown again.
 
 You can generate multiple client secrets for your client ID.  For example, you can create multiple secrets for multiple projects.
 
+When using v2 of this REST Publish API, the ClientId and API Keys are automatically re-generated, by backend services.<!-- todo: every time you use any endpoint of the REST API?  what action causes re-generation of ClientId and API Keys? -->
```

Comment:
when are ClientId and API Keys auto re-generated - every time you use any endpoint of the REST API? what action causes re-generation of ClientId and API Keys?  

orig wording said "for every developer" but removed that wording b/c seemed to not address key point or key distinction.
seems superfluous to say "system works this way for every dev" instead of simply "system works this way".  main point is, what Dev action triggers regen?

Reply:
closing Outdated comments.  added todo notes in .md; will create fresh Comment at each todo note